frontend/src/app/features/details/parcelDescriptions/parcelDescriptions.css

```diff
@@ -0,0 +1,4 @@
+.custom-land-description-disabled-input {
+    background-color: #E9E6E4 !important;
+    color: #A3A2A0 !important;
+}
```

Comment:
I did a quick codebase search for these colors and looks like this is the only place that uses them. Is this a special use case where the default "disabled" styling doesn't cut it?

Reply:
ok, I tested some of the table controls locally and looks like simply adding `isDisabled` prop doesn't apply any styling at all. The control is disabled functionally, but not visually.

I vote for removing these css rules for now:
1. It's an issue that's way out of scope of this PR and should be handled centrally on the shared components
2. In most cases we shouldn't use hard-coded colors and instead rely on those provided in `@bcgov/design-tokens` package

Reply:
That's fine. I just used the dropper to take the colors from figma.